lib/trento/infrastructure/router.ex

```diff
@@ -19,6 +19,7 @@ defmodule Trento.Router do
     RollUpHost,
     RollUpSapSystem,
     SelectChecks,
```

Comment:
As we have now a checks selection for hosts and clusters, should we maybe rename SelectChecks to SelectClusterChecks ? :D

Reply:
Yes, we might want to align naming, however let's defer it.